bcftools.wdl

```diff
@@ -127,6 +127,7 @@ task Sort {
     input {
         File inputFile
         String outputPath = "output.vcf.gz"
+        String tmpDir = "./sorting-tmp"
```

Comment:
new input needs to be added to parameter_meta as well

Reply:
Thank you. I added it in the parameter meta.